mam/v2/mam/mam.h

```diff
@@ -26,11 +26,10 @@
 #include "mam/v2/wots/wots.h"
 
 typedef struct mam_ialloc_s {
-  ialloc *a; /*!< Context passed to `create_sponge` and `destroy_sponge`. */
-  sponge_t *(*create_sponge)(
-      ialloc *a); /*!< Allocator for sponge interfaces used by
-                     channels/endpoints (WOTS, PB3 sponge, PB3 fork sponge). */
-  void (*destroy_sponge)(ialloc *a, sponge_t *); /*!< Deallocator. */
+  sponge_t *(*create_sponge)(); /*!< Allocator for sponge interfaces used by
+                                              channels/endpoints (WOTS, PB3
```

Comment:
Indentation is required for C comments.

Reply:
According to our formatting rules, this is ok.
Have a look at https://github.com/iotaledger/entangled/blob/develop/.clang-format.
However, all documentation will be rewritten with our documentation style so this will disappear anyway.